CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+## How to contribute
+
+If you are interested in joining the project, then please review our [Projects Board](https://github.com/hackforla/VRMS/projects/2) to see if anything interests you. If so, then please do the following.
+
+1. Review the Hack for LA Getting Started Guide https://www.hackforla.org/getting-started.
+
+1. Join the VRMS weekly meetings https://www.vrms.io/.
+
+1. Reach out to our project leads to discuss how you would like to contribute.
```

Comment:
I think it should be the same or almost the same as the first paragraph of https://github.com/hackforla/VRMS/wiki/Join-the-Team 
The wiki page was updated by Adam recently.

Reply:
👍